### PR TITLE
Firmware imu: Wait after baudrate change command for new baudrate to get applied

### LINKED ISSUE #89: [FRIMWARE/SDK] IMU reports sometimes only zeros
In ~1/4 of the cases lately the IMU returns only zeros. I am not sure what is going on but assume that something with setting up the IMU goes wrong and the master board is not receiving any packages from the IMU.

Has someone else observed this?

@thomasfla any idea how this could be fixed or did you observe this as well?

### PR DESCRIPTION
**Pull request by jviereck**
Fixes #89.

### CODE REVIEW

**Comment by jviereck** [comment]
I've tested a version of this code on solo12 for the last days. The IMU is always reporting data in comparison to the state reported in #89.

File: firmware/components/uart_imu/uart_imu.c [hunk 1]
```diff
@@ -247,32 +247,34 @@ int imu_init()
   const char cmd5[8] = {0x75, 0x65, 0x01, 0x02, 0x02, 0x06, 0xE5, 0xCB};
   const char cmd6[13] = {0x75, 0x65, 0x0C, 0x07, 0x07, 0x40, 0x01, 0x00, 0x0E, 0x10, 0x00, 0x53, 0x9D}; // 921600 bauds
 
-  printf("sending new baud rate setting to the IMU \n");
-  vTaskDelay(30);
   uart_write_bytes(UART_NUM, cmd0, sizeof(cmd0));
-  vTaskDelay(10);
+
+  printf("sending new baud rate setting to the IMU \n");
```
Comment (MaximilienNaveau) [inline firmware/components/uart_imu/uart_imu.c:252]: to be removed

**Review by MaximilienNaveau — APPROVED** [review]
some small debug info to be removed

File: firmware/components/uart_imu/uart_imu.c (hunk 1 above)
Comment (jviereck) [reply firmware/components/uart_imu/uart_imu.c:252]: This message was in the original code as well (see the diff). I can remove it but as it was there before, I thought to keep it.

**Pull request merged**